packages/delegation-process/src/utilites/errorMappers.ts

```diff
@@ -38,3 +39,12 @@ export const getDelegationErrorMapper = (error: ApiError<ErrorCodes>): number =>
   match(error.code)
     .with("delegationNotFound", () => HTTP_STATUS_NOT_FOUND)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const approveRejectDelegationErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("delegationNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with("operationRestrictedToDelegate", () => HTTP_STATUS_FORBIDDEN)
+    .with("incorrectState", () => HTTP_STATUS_BAD_REQUEST)
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
```

Comment:
We really like to have one mapper to handle two endpoints errors?

Reply:
Right now they are the same, why duplicate code? If needed we can always split

Reply:
Can I suggest to do something like this?

`export const approveDelegationErrorMapper = rejectDelegationErrorMapper;`

In this case each one has their specific mapper but since they are the same we don't duplicate code

Reply:
I'm all for one error mapper for each route, I would not renounce flexibility for such a small repetition